hwdb.d/60-keyboard.hwdb

```diff
@@ -1931,6 +1931,13 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnViewSonic:pnVPAD10:*
  KEYBOARD_KEY_db=prog1                                  # Home button
  KEYBOARD_KEY_dd=back                                   # Second button labeled Back
 
+###########################################################
+# Positivo-Vaio
+###########################################################
+# Vaio FE14
+evdev:name:AT Translated Set 2 keyboard:dmi:bvnPositivoTecnologiaSA:bvr*:bd*:svnPositivoBahia-VAIO:pnVJFE41F11*:pvr*:*
```

Comment:
I guess, `bvn` field can be safely `bvn*`.

Reply:
I tested the modification and still working good. :) I'll make the changes.

Reply:
I tested the modification and still working good. :) I'll make the changes.